library/ghost

```diff
@@ -4,14 +4,14 @@ Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
              Joseph Ferguson <yosifkit@gmail.com> (@yosifkit)
 GitRepo: https://github.com/docker-library/ghost.git
 
-Tags: 1.17.2, 1.17, 1, latest
+Tags: 1.17.3, 1.17, 1, latest
 Architectures: amd64, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: f7fe8c847bce4ebcfde5bea23a8c45c7a242e243
+GitCommit: d0f88d93e06cdca5ea84cc3bbb13108735bf6a00
```

Comment:
It seems there was another commit right when you opened this PR to update ghost-cli [1.3.0](https://github.com/TryGhost/Ghost-CLI/releases/tag/1.3.0) (https://github.com/docker-library/ghost/commit/8ddece4dde5727e3324198d86ae565511ec2fbef).  Do you want to update the PR or leave it as is?  I am fine either way, we'll probably make an "update all images" PR later today anyway.